src/main.c

```diff
@@ -71,31 +71,31 @@ static void usage(int code, int keep_it_short) {
       "Command options:\n"
       "  -n, --null-input          use `null` as the single input value;\n"
       "  -R, --raw-input           read each line as strings, not parsing as JSON;\n"
-      "  -s, --slurp               read all inputs into an array and use it as the single input value;\n"
```

Comment:
I think it'd be better to retain the original text and add a `\n` and some `\t`s to indent the continuation correctly.

Reply:
Ok! do you know if multi line is ok with ";", i think that is some kind of auto completion convension?

@itchyny do you know?

Reply:
Better now?